backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -378,23 +376,25 @@ export class ContributionResolver {
     return result
   }
 
-  @Authorized([RIGHTS.LIST_UNCONFIRMED_CONTRIBUTIONS])
-  @Query(() => ContributionListResult) // [UnconfirmedContribution]
-  async adminListAllContributions(
+  @Authorized([RIGHTS.ADMIN_LIST_CONTRIBUTIONS])
+  @Query(() => ContributionListResult)
+  async adminListContributions(
     @Args()
     { currentPage = 1, pageSize = 3, order = Order.DESC }: Paginated,
     @Arg('statusFilter', () => [ContributionStatus], { nullable: true })
     statusFilter?: ContributionStatus[],
+    @Arg('userId', () => Int, { nullable: true })
+    userId?: number,
   ): Promise<ContributionListResult> {
-    const [dbContributions, count] = await findContributions(
+    const [dbContributions, count] = await findContributions({
       order,
       currentPage,
       pageSize,
-      true,
-      ['user'],
-      undefined,
+      withDeleted: true,
+      userId,
+      relations: ['user'],
```

Comment:
I think that we also need messages hier, so that admin get message count of contributions.
```suggestion
      relations: ['user', 'messages'],
```

Reply:
You did this in your PR. So I am not doing this here.